README.md

```diff
@@ -9,3 +9,664 @@ ExpressRouterAdapter is available as a package on NPM for use with a module bund
 ```shell
 npm install --save @symbiotic/express-router-adapter
 ```
+
+## Getting started
+
+ExpressRouterAdapter enhances express, so lets start with a simple express app. This is not a deep dive intro express, see express documentation.
+
+```js
+
+const app = express();
+
+app.get('/pets/:petId', (req, res) => {
+  res.json({ petId: req.params.petId, name: 'Sawyer '});
+});
+
+app.listen(4000);
+
+```
+
+Lets install ExpressRouterAdapter and dependncies
```

Comment:
typo, dependncies -> dependencies

Reply:
We migth want to consider moving the 'We are using IOC' to the above sentence since we are saying 'and dependencies' even though aurelia-dependency-injection is optional